f2/src/components/formik/link/index.js

```diff
@@ -2,6 +2,7 @@ import styled from '@emotion/styled'
 import { color } from 'styled-system'
 
 export const A = styled.a`
-  ${color};
+  color: ${(props) => (props.color ? props.color : color)} !important;
```

Comment:
If there is no color in props, what's the color then?
Why would we need !important. Who and where is overriding our code. I indeed find that color itself is not effective.

Reply:
Or it should be this?
color: ${(props) => (props.color ? props.color : null)} !important;

Reply:
!important is needed because a bootstrap css rule is overriding this. I have changed the color prop to be:

`color: ${(props) => (props.color ? props.color : null)} !important;`